src/style/app.module.css

```diff
@@ -992,6 +992,181 @@ hr {
   background: #afffe8 !important;
 }
 
+.organizationFundCampaignContainer {
+  margin: 0.5rem 0;
+}
+
+.rowBackgroundOrganizationFundCampaign {
+  background-color: var(--bs-white);
+  max-height: 120px;
+}
+
+.campaignModal {
+  max-width: 80vw;
+  margin-top: 2vh;
+  margin-left: 13vw;
+}
+
+.greenregbtnOrganizationFundCampaign {
+  margin: 1rem 0 0;
+  margin-top: 15px;
+  border: 1px solid var(--bs-gray-300);
+  box-shadow: 0 2px 2px #e8e5e5;
+  padding: 10px 10px;
+  border-radius: 5px;
+  background-color: #31bb6b;
+  width: 100%;
+  font-size: 16px;
+  color: white;
+  outline: none;
+  font-weight: 600;
+  cursor: pointer;
+  transition:
+    transform 0.2s,
+    box-shadow 0.2s;
+  width: 100%;
+  flex: 1;
+}
+
+.goalButtonOrganizationFundCampaign {
+  border: 1px solid #31bb6b !important;
+  color: #31bb6b !important;
+  width: 75%;
+  padding: 10px;
+  border-radius: 8px;
+  display: block;
+  margin: auto;
+  box-shadow: 5px 5px 4px 0px rgba(49, 187, 107, 0.12);
+}
+
+.redregbtn {
+  margin: 1rem 0 0;
+  margin-top: 15px;
+  border: 1px solid #e8e5e5;
+  box-shadow: 0 2px 2px #e8e5e5;
+  padding: 10px 10px;
+  border-radius: 5px;
+  width: 100%;
+  font-size: 16px;
+  color: white;
+  outline: none;
+  font-weight: 600;
+  cursor: pointer;
+  transition:
+    transform 0.2s,
+    box-shadow 0.2s;
+  width: 100%;
+  flex: 1;
+}
+
+.campaignNameInfo {
+  font-size: medium;
+  cursor: pointer;
+}
+.campaignNameInfo:hover {
+  color: blue;
+  transform: translateY(-2px);
+}
+
+.inputFieldOrganizationFundCampaign {
+  background-color: white;
+  box-shadow: 0 1px 1px #31bb6b;
```

Comment:
1. We are migrating away from the green theme as part of our support for color blind users.
2. Take cues from other similar elements in this file and the PR mentioned in the issue.
    1. https://github.com/PalisadoesFoundation/talawa-admin/pull/2466